packages/fileeditor-extension/src/commands.ts

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import { WidgetTracker } from '@jupyterlab/apputils';
+
+import { IConsoleTracker } from '@jupyterlab/console';
+
+import { IDocumentWidget } from '@jupyterlab/docregistry';
+
+import { FileEditor } from '@jupyterlab/fileeditor';
+
+import {
+  IEditMenu,
+  IFileMenu,
+  IMainMenu,
+  IRunMenu,
+  IViewMenu
+} from '@jupyterlab/mainmenu';
+
+import { CommandRegistry } from '@phosphor/commands';
+
+import { JSONObject } from '@phosphor/coreutils';
+
+import { Menu } from '@phosphor/widgets';
+
+import { CommandIDs } from './index';
+
+export default class Commands {
+  static addMenuItems(
```

Comment:
I would probably take these out of the class and make each menu addition a separate function, then perhaps one overall function that calls each smaller function. That way another extension could simply import and add whichever parts they wanted, or they could add everything.

Reply:
I can make thing even more modularized, no problem for me. I'll try to make those changes and push them either Monday or Tuesday